README.md

```diff
@@ -74,34 +73,39 @@ You'll need a [compatible GPU](https://docs.modular.com/max/faq#gpu-requirements
    ```
 3. Install a package manager to run the Mojo🔥 programs:
 
-    ### Option 1: [`uv`](https://docs.astral.sh/uv/getting-started/installation/) (recommended for users)
```

Comment:
Thanks! why changing the readme format? The changes aren't ideal. Please bring those back.

Reply:
Yeah it was just autoformatting, I reverted it